roles/pulp_redis/tasks/main.yml

```diff
@@ -29,9 +29,18 @@
 
     - name: Install Redis
       package:
-        name: redis
+        name: '{{ pulp_redis_package_name }}'
         state: present
 
+    - name: Enable Redis SCL
```

Comment:
What happens when an existing install received Redis on their system from non-SCL and on the next run they receive another Redis from the SCL?

Reply:
In this scenario, if one decides to go this way, some manual intervention would be required to at least stop and disable the service (already installed) to start on boot. Else the "newer" install will refuse to start because it won't be able to bind when using TCP for example.

Similarly same manual steps would be required if one goes from `rh-redis5` to `rh-redis6` SLCs since services name are different. There is nothing much we can (and IMO should do) do here if not try to guess user system configuration. (ie. List all units, grep the one that matches `redis`, ensure they are all disabled but the one specified by `pulp_redis_server_name`).

While do-able I feel it clutters the role and put us in a "I handle it all" situation where I feel system administrator should be mindffull of those behavior/patterns.

Reply:
Agreed that the previously installed non-SCL redis service needs to be stopped or the new one will fail to start. Agreed on the approach of not trying to have it handle all possible scenarios.